src/components/NewExperiment.vue

```diff
@@ -1595,6 +1714,10 @@ export default Vue.extend({
         .then(response => {
           const mnxMetabolites: MetaNetXMetabolite[] = response.data;
           return mnxMetabolites.map((mnxMetabolite, index) => {
+            // Handle pasting empty cells
+            if (strs[index] === "") {
+              return null;
+            }
```

Comment:
This may be unreliable

Check that both batch mnx and uniprot results don't get misaligned.
`query=4,,1,3` and `uploadQuery=4  1 3` would have to return `[{id: 4}, anything, {id: 1}, {id: 3}]`, but I'd expect `[{id: 4}, {id: 1}, {id: 3}]`

Reply:
Metanetx and Uniprot services behave differently:
1. mnx - returns empty object if no match was found (it also checks EC number and when we are sending empty string it can return any object which has EC equals to `''`). We are iterating through response data safely when mapping and handling pasting empty cells
2. uniprot - doesn't return anything if no match was found. That's why we are iterating through initial query

Reply:
> it also checks EC number and when we are sending empty string it can return any object which has EC equals to `''`

Should we change that to disallow search with empty string? For example by returning HTTP 400, or just leave it `null`?

Reply:
I think it's better with `null`, so pasting empty cells will clear the fields as we discussed